### ISSUE TITLE
Overriding deprecated methods

### ISSUE THREAD

**Issue by tomlut** [mawinkler/astroweather#15]
After updating to core-2022.7.0b0 the following warning is present:

```
Logger: homeassistant.components.weather
Source: components/weather/__init__.py:308
Integration: Weather (documentation, issues)
First occurred: 19:16:37 (2 occurrences)
Last logged: 19:16:40

custom_components.astroweather.weather::AstroWeatherWeather is overriding deprecated methods on an instance of WeatherEntity, this is not valid and will be unsupported from Home Assistant 2023.1. Please report it to the custom component author.
```

Possibly related to this developer blog entry:

https://developers.home-assistant.io/blog/2022/06/27/weather_entity_refactoring/

**Comment by mawinkler** [comment]
Hi @tomlut, thank you for reporting this. Will take care of this with the next update.

**Comment by mawinkler** [comment]
Should now be fixed with [v0.22.0](https://github.com/mawinkler/astroweather/tree/v0.22.0)

**Comment by tomlut** [comment]
It is, thanks!
